test/e2e/e2e_test.go

```diff
@@ -225,6 +226,41 @@ var _ = Describe("Zero Trust Workload Identity Manager", Ordered, func() {
 			By("Waiting for SPIRE Agent DaemonSet to become Available")
 			utils.WaitForDaemonSetAvailable(testCtx, clientset, utils.SpireAgentDaemonSetName, utils.OperatorNamespace, utils.DefaultTimeout)
 		})
+
+		It("SPIFFE CSI Driver should be deployed successfully by creating a SpiffeCSIDriver object", func() {
+			By("Creating SpiffeCSIDriver object")
+			spiffeCSIDriver := &operatorv1alpha1.SpiffeCSIDriver{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "cluster",
+				},
+				Spec: operatorv1alpha1.SpiffeCSIDriverSpec{},
+			}
+
+			By("Debug: Printing SpiffeCSIDriver object details")
+			klog.V(4).Infof("BEFORE Create - Name: %s\n", spiffeCSIDriver.Name)
+			klog.V(4).Infof("BEFORE Create - Namespace: '%s' (empty for cluster-scoped)\n", spiffeCSIDriver.Namespace)
+			klog.V(4).Infof("BEFORE Create - Spec: %+v\n", spiffeCSIDriver.Spec)
```

Comment:
nit: just a suggestion, do we need multiple statements?
```suggestion
			klog.V(4).Infof("BEFORE Create - \nName: %s\nNamespace: '%s\nSpec: %+v\n", spiffeCSIDriver.Name, spiffeCSIDriver.Namespace, spiffeCSIDriver.Spec)
```